py-polars/Makefile

```diff
@@ -25,11 +25,11 @@ fmt: venv  ## Run autoformatting and linting
 	$(VENV_BIN)/isort .
 	$(VENV_BIN)/black .
 	$(VENV_BIN)/blackdoc .
-	$(VENV_BIN)/pyupgrade --py37-plus `find polars/ tests/ -name "*.py" -type f`
+	$(VENV_BIN)/pyupgrade --py37-plus `find polars docs tests tests_parametric -name "*.py" -type f`
```

Comment:
`tests_parametric` doesn't exist anymore! It's now part of the `tests` folder.

Reply:
Ah, the directory was still visibible on my system, but the files have moved indeed. Thanks for spotting.